File: lab_5/lab_5.py
def rev_num(num):
    """returns reversed integer"""
    rev_num = 0
    while num > 0:
        rem = num % 10
        rev_num = rev_num*10 + rem
        num = num // 10
    return rev_num

def main():
    for num in range(1000000, 1000000001):
        if num == rev_num(num):
            print(num)

main()

File: Lab5/Lab5.cpp
#include <iostream>

using namespace std;

int reversedNumber(int);

int main() {
	for (int num = 1000000; num <= 1000000000; num++)
	{
		if (num == reversedNumber(num))
		{
			cout << num << "\n";
		}
	}

	return 0;
}

/*
* returns reversed integer
*/
int reversedNumber(int num) {
	int revNum = 0, rem;
	while (num > 0)
	{
		rem = num % 10;
		revNum = revNum * 10 + rem;
		num = num / 10;
	}
	return revNum;
}